.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
+on: [push, pull_request]
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: Check out code
+        uses: actions/checkout@v1
```

Comment:
Something interesting, maybe for some other time: https://github.com/actions/cache/blob/main/examples.md#go---modules